ui/imports/shared/controls/AmountInput.qml

```diff
@@ -11,7 +11,7 @@ Input {
     id: root
 
     property int maximumLength: 10
-    property var locale: Qt.locale()
+    property var locale: LocaleUtils.userInputLocale
```

Comment:
Would it make sense to use type 'Locale' instead of 'var' ?
https://doc.qt.io/qt-5/qml-qtqml-locale.html#details

Reply:
well... it's safer this way; also in general can by overridden with "anything" when being mocked, ie under Storybook